src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -138,6 +138,7 @@ export default class QuerySummary extends React.Component<{ routingStore:Extende
     @computed get queryForm(){
         return <div style={{margin:"10px -20px 0 -20px"}}>
             <QueryAndDownloadTabs onSubmit={this.onSubmit}
+                                  forkedExperience={false}
```

Comment:
I find it vague, can you rename this prop, maybe "bigStudyViewButton" or something?

Reply:
What about forkedMode?  I actually think "forkedXXX" is more descriptive.  It at least hints at the concept behind this mode.   If we can come up with a better name than "forked" I'm open to it.   I will change to mode because it's better than "experience" for sure.

Reply:
sure, that works for me